packages/perseus/src/mixins/widget-jsonify-deprecated.ts

```diff
@@ -1,23 +0,0 @@
-/* Free implementation of getUserInput. This should be used sparingly, since it
- * just returns all the widget's props rather than picking out those which were
- * input by the user.
- */
-import _ from "underscore";
-
-import WIDGET_PROP_DENYLIST from "./widget-prop-denylist";
```

Comment:
I'm guessing we can't _quite_ delete this file yet, right? `widget-prop-denylist.ts` I mean.

Reply:
Looks like it's used by a few things, but I noted in our Futures document to investigate if it could be removed.